Aqovia.Utilities.SagaMachine/SagaMachine.cs

```diff
@@ -30,7 +30,7 @@ public SagaMachine(IKeyValueStore keyValueStore, Func<IEnumerable<ISagaMessageId
 
         public async Task Handle<TMessage>(TMessage message) where TMessage : ISagaMessageIdentifier
         {
-            await Task.Run(() => ProcessMessage(message));
+            await Task.Run(() => ProcessMessage(message)).ConfigureAwait(false);
```

Comment:
I must say that I also found it initially pretty stupid for .NET to introduce a nice single `await` keyword, only to tell everyone that by default it should be used with a boilerplate `.ConfigureAwait(false)`.

After reading [this article](http://blogs.msdn.com/b/pfxteam/archive/2012/06/15/executioncontext-vs-synchronizationcontext.aspx) i now at least understand that this is not the 'await' keyword's fault, but rather a design flaw of .NET (read near _'Isn’t SynchronizationContext part of ExecutionContext?'_).

It's just too bad that instead of allowing this design flaw to be fixed in a single place for those who don't depend on the behaviour using something like `[assembly:ShouldFlowSynchronizationContextByDefault(false)]`, they resort to recommend adorning every use of `await` with some boilerplate.